dimod/core/composite.py

```diff
@@ -75,10 +77,14 @@ def child(self):
             raise RuntimeError("A Composite must have at least one child Sampler")
 
 
+
 class ComposedSampler(Sampler, Composite):
```

Comment:
I think if you change this to `class ComposedSampler(Composite, Sampler):` then you don't need to implement the method on this class.